client/components/room/Video.jsx

```diff
@@ -2,17 +2,23 @@ import React from 'react';
 import io from 'socket.io-client';
 import webrtc from 'webrtc-adapter';
 
-// const port = process.env.PORT || 3000;
 const server = location.origin;
-// const server2 = 'https://tailbud-pr-17.herokuapp.com/';
 const socket = io(server);
 
 let localStream;
-let pc1;
-let pc2;
-let offerOptions = {
-  offerToReceiveAudio: 1,
-  offerToReceiveVideo: 1
+let remoteStream;
+let turnReady;
+let pc;
+
+const configuration = {
+  'iceServers': [{
+    'urls': 'stun:stun.l.google.com:19302'
+  }]
+};
+
+let sdpConstraints = {
```

Comment:
In a later pull, could we get some small comments that give explanation for what these lingo-heavy sections do?